### DISCUSSION TITLE
FTXT sent to terminal with formatting

### DISCUSSION THREAD

**Discussion by fmw42** [ImageMagick/ImageMagick#5165] [Help]
Sorry, I am confused about sending formatted text to the terminal using ftxt.

This works fine:

```
magick xc:"sRGB(10%,20%,30%)" "ftxt:-"
0,0:6553.5,13107,19660.5
```

But I get nothing returned from the following trying to just get the hex value:

```
magick xc:"sRGB(10%,20%,30%)" "ftxt:format=\H:-"

nor

magick xc:"sRGB(10%,20%,30%)" "ftxt:format=\H"

```

Please correct my syntax.

**Comment by snibgo** [comment]
The format goes into a define, not the output filename:
```
f:\web\im>%IMG7%magick xc:"sRGB(10%,20%,30%)" -define "ftxt:format=\H" ftxt:-

#199A33334CCD
```

**Reply by fmw42** [reply]
Thanks
